From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rubens=20Brand=C3=A3o?= <git@rubens.io>
Date: Mon, 13 Mar 2023 10:42:57 -0300
Subject: [PATCH] 5093: Sleigh 68000 CPUs exports a length 0 value

A value with length 0 should be impossible, but also, all constructors from a table should export a value with the same length.
---
 Ghidra/Processors/68000/data/languages/68000.sinc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Ghidra/Processors/68000/data/languages/68000.sinc b/Ghidra/Processors/68000/data/languages/68000.sinc
index 5e59bde3e..447d6ddac 100644
--- a/Ghidra/Processors/68000/data/languages/68000.sinc
+++ b/Ghidra/Processors/68000/data/languages/68000.sinc
@@ -2203,7 +2203,7 @@ fcc: "un"	is fcode=0x08			{ tmp:1 = $(NAN_FP); clearflags_fp(); export tmp; }
 
 fcc: "f"	is fcode=0x00			{ export 0:1; }
 fcc: "t"	is fcode=0x0f			{ export 1:1; }
-fcc: "sf"	is fcode=0x10			{ export 0:0; }
+fcc: "sf"	is fcode=0x10			{ export 0:1; }
 fcc: "st"	is fcode=0x1f			{ export 1:1; }
 fcc: "seq"	is fcode=0x11			{ tmp:1 = $(Z_FP); clearflags_fp(); export tmp; }
 fcc: "sne"	is fcode=0x1e			{ tmp:1 = $(Z_FP); clearflags_fp(); export tmp; }
-- 
2.39.1

